cloudinit/cmd/status.py

```diff
@@ -163,82 +151,80 @@ def get_parser(parser=None):
     return parser
 
 
-def handle_status_args(name, args) -> int:
-    """Handle calls to 'cloud-init status' as a subcommand."""
-    # Read configured paths
-    paths = read_cfg_paths()
-    details = get_status_details(paths, args.wait)
-    if args.wait:
-        while details.status in (
-            UXAppStatus.NOT_RUN,
-            UXAppStatus.RUNNING,
-            UXAppStatus.DEGRADED_RUNNING,
-        ):
-            if args.format == "tabular":
-                sys.stdout.write(".")
-                sys.stdout.flush()
-            details = get_status_details(paths, args.wait)
-            sleep(0.25)
-    details_dict: Dict[str, Union[None, str, List[str], Dict[str, Any]]] = {
+def translate_status(
+    running: RunningStatus, condition: ConditionStatus
+) -> Tuple[str, str]:
+    """Translate running and condition status to human readable strings.
+
+    Returns (status, extended_status).
+    Much of this is for backwards compatibility
+    """
+    # If we're done and have errors, we're in an error state
+    if condition == ConditionStatus.ERROR:
+        return ("error", f"{condition.value} - {running.value}")
```

Comment:
style question: Was it intentional to use parenthesis on this line and the next return but not the final one? I personally prefer never using them when returning tuples, but I'm curious if there is some stylistic reason for doing it this way?

Reply:
Nope, just lack of conscious thought. I was also notorious at a previous company for mixing double quotes and single quotes in the same file/function, but thankfully black has made that a non-issue for me. :smile: 

I removed the parens from these.